test/files/run/repl-no-imports-no-predef.scala

```diff
@@ -0,0 +1,112 @@
+object Test extends scala.tools.partest.ReplTest {
+
+  override def transformSettings(settings: scala.tools.nsc.Settings) = {
+    settings.noimports.value = true
+    settings.nopredef.value = true
+    settings
+  }
+
+  // replace indylambda function names by <function0>
+  override def eval() = {
+    val lines = super.eval
+    val r = """\$\$Lambda.*""".r
+    lines.map(l => r.replaceAllIn(l, "<function0>"))
```

Comment:
You can override `normalize` https://github.com/scala/scala/blob/2.12.x/src/partest-extras/scala/tools/partest/ReplTest.scala#L24